internal/core/api.rs

```diff
@@ -301,6 +302,21 @@ pub enum CloseRequestResponse {
     KeepWindowShown,
 }
 
+/// This enum describes whether a virtual should be shown or closed.
+#[derive(Copy, Clone, Debug, PartialEq, Default)]
```

Comment:
I think it needs to be non_exhaustive.
And it doesn't need to have a Default.
Also not Copy